acceptance-tests/src/test/java/io/blueocean/ath/live/GithubEditorTest.java

```diff
@@ -129,7 +129,7 @@ public void testEditor() throws IOException {
         sseClient.clear();
         BranchPage branchPage = activityPage.clickBranchTab();
         branchPage.openEditor("master");
-        editorPage.saveBranch("new-branch");
+        editorPage.saveBranch("new - branch");
```

Comment:
Hrm - I think that the test for this should be explicit in this file - ie the EditorPage.java stays as it was, and first it "tries" with spaces, gets exception, and then tries without.